include/swift/AST/Decl.h

```diff
@@ -5863,6 +5863,9 @@ enum class PropertyWrapperSynthesizedPropertyKind {
 class VarDecl : public AbstractStorageDecl {
   friend class NamingPatternRequest;
   NamedPattern *NamingPattern = nullptr;
+  /// The opened element environment of the pack expansion to use as
+  /// the variable's context generic environment when the variable is
+  /// declared in context of a for-in loop over the elements of a parameter pack.
   GenericEnvironment *OpenedElementEnvironment = nullptr;
```

Comment:
This is a mouthful. Could we flip the sentence around "when" to make it easier to read?